model/src/main/proto/topic.proto

```diff
@@ -88,6 +88,9 @@ message ChapterSummary {
 
   // The thumbnail corresponding to this chapter.
   LessonThumbnail chapter_thumbnail = 5;
+
+  // Prerequisite chapter to unlock this chapter.
```

Comment:
Suggest adding a note here that this is only defined when the play state is ``NOT_PLAYABLE_MISSING_PREREQUISITES``.

I suggest also calling it ``missing_prerequisite_chapter`` to make this more clear in code, as well. There's technically always a prereq chapter except for the first in a story, but we really want to emphasize this is a case where the prerequisite is missing.

Reply:
Done.